### PR TITLE
Fix withdraw reasons decoding

### LINKED ISSUE #131: Type missed when calling api.query.genericAsset.locks function
Get type missed error when calling `api.query.genericAsset.locks` function
```
b = await api.query.genericAsset.locks('5FCdFHNPR62USs3xeA8Pnj2D87n4RLG1xRB2jS4tbcs7t19j')
Unable to decode storage genericAsset.locks: createType(Vec<BalanceLock>):: Unable to create Enum via index 31, in Fee, Misc, All
2020-07-27 18:46:50        RPC-CORE: subscribeStorage(keys: Vec<StorageKey>): StorageChangeSet:: createType(Vec<BalanceLock>):: Unable to create Enum via index 31, in Fee, Misc, All
```

### PR DESCRIPTION
**Pull request by holygits**
Closes #131 

Adds:
- BalanceLock and WithdrawReasons types

Changes:
- Applied formatting to some files involved

### CODE REVIEW

File: packages/api/test/e2e/queries.e2e.ts [hunk 1]
```diff
@@ -34,16 +32,12 @@ describe('e2e queries', () => {
     api = await initApiPromise();
   });
 
-  afterAll(async done => {
-    if (api) {
-      return await api.disconnect();
-    }
-    api = null;
-    done();
+  afterAll(async () => {
```
Comment (holygits) [inline packages/api/test/e2e/queries.e2e.ts:35]: this was always forcing a 60s timeout before 😅

Comment (benxgao) [reply packages/api/test/e2e/queries.e2e.ts:35]: What is the timeout right now?

File: packages/types/src/runtime/ga/WithdrawReasons.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  // Whether the withdraw reasons include transaction tips.
+  isTip(): boolean {
+    return (this.toNumber() & ReasonMask.TIP) === ReasonMask.TIP;
+  }
+  // Whether the withdraw reasons include transaction fee payment.
+  isTransactionPayment(): boolean {
+    return (this.toNumber() & ReasonMask.TRANSACTION_PAYMENT) === ReasonMask.TRANSACTION_PAYMENT;
+  }
+  // Whether the withdraw reasons include transfers.
+  isTransfer(): boolean {
```
Comment (benxgao) [inline packages/types/src/runtime/ga/WithdrawReasons.ts:57]: How about something like `isTransferingOwnership`?

File: packages/types/src/runtime/ga/WithdrawReasons.ts [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  // Whether the withdraw reasons are 'all' of them
+  isAll(): boolean {
+    return this.toNumber() === WithdrawReasons.all();
+  }
+  // Whether the withdraw reasons include higher-level fees.
+  isFee(): boolean {
+    return (this.toNumber() & ReasonMask.FEE) === ReasonMask.FEE;
+  }
+  // Whether the withdraw reasons include reservation.
+  isReserve(): boolean {
```
Comment (benxgao) [inline packages/types/src/runtime/ga/WithdrawReasons.ts:45]: How about something like `isReservingBalance` or `isReservingFund`?

File: packages/api/test/e2e/queries.e2e.ts (hunk 1 above)
Comment (holygits) [reply packages/api/test/e2e/queries.e2e.ts:35]: 60s configured in jest.init.js

**Pull request merged**